.github/workflows/deploy-to-aws.yml

```diff
@@ -88,6 +94,18 @@ jobs:
         shell: bash
         working-directory: ${{ inputs.working-directory }}
     environment: ${{ (inputs.environment && ((github.ref == 'refs/heads/main') && 'AWS_PROD' || 'AWS_TEST')) || '' }}
+    env:
```

Comment:
These secrets were directly on the deploy step so that they would not be accessible elsewhere. I guess we'll have to decide on the trade-off of having them duplicated across two steps or exposing them to all steps.

Reply:
ah, yes actually that a very good reason to duplicate them ... I'll fix that @voj